CASMPET-6408: Add troubleshooting docs for Nexus-Keycloak intergration (#4047)
Cray-HPE/docs-csm: operations/package_repository_management/Troubleshoot_Nexus.md

- [`lookup registry.local: no such host`](#lookup-registrylocal-no-such-host)
- [`Error initiating layer upload` ... `in registry.local`](#error-initiating-layer-upload--in-registrylocal)
- [`error: not ready: https://packages.local`](#error-not-ready-httpspackageslocal)

> - [Unable to log in with Keycloak accounts](#unable-to-login-with-keycloak-accounts)

Reviewer: ```suggestion
- [Unable to log in with Keycloak accounts](#unable-to-log-in-with-keycloak-accounts)
```

> - [`Unable to login with Keycloak accounts`](#unable-to-login-with-keycloak-accounts)

Reviewer: Only need to enclose it in backticks if we are quoting the exact error message being seen by the user, which I don't think is the case here.

```suggestion
- [Unable to login with Keycloak accounts](#unable-to-login-with-keycloak-accounts)
```

deployment and additional diagnosis is required. In this case, the current
Nexus deployment probably needs to be uninstalled and the `nexus-data` PVC
removed before attempting to deploy again.

## Unable to login with Keycloak accounts

> First ensure the Keycloak account that is attempting to login has proper Nexus permissions.

Reviewer: The HPE style guide dictates using "login" as a noun and "log in" as a verb.
```suggestion
First ensure the Keycloak account that is attempting to log in has proper Nexus permissions.
```

> See [Manage Repositories with Nexus](./Manage_Repositories_with_Nexus.md) for information

Reviewer: ./ is superfluous in this context. In scripts it makes sense, because of PATH issues, but not in a link context.
```suggestion
See [Manage Repositories with Nexus](Manage_Repositories_with_Nexus.md) for information
```

on what Keycloak permissions they need.

> If the account has all the proper permissions and still fails to login to Nexus

Reviewer: ```suggestion
If the account has all the proper permissions and still fails to log in to Nexus, then
```
